### PR TITLE
Patch hidden packs issue

### PR DESCRIPTION
**Pull request by AradCarmi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by kgal-pan** [comment]
Affecting https://github.com/demisto/content-private/pull/595 (comment for reference)

**Comment by AradCarmi** [comment]
Force merge was needed due to unrelated timeout errors in the trigger-private-build job.
Also, this PR is a fix to this [issue](https://github.com/demisto/content-private/runs/7060943124?check_suite_focus=true#step:44:91) and secondly to the trigger-private-build issue.

**Comment by Noy-Maimon** [comment]
I see this error in the private build - what is it related to? 
image

**Comment by yaakovi** [comment]
> I see this error in the private build - what is it related to? image

this is coming from the test-module of the integration

File: Tests/configure_and_test_integration_instances.py [hunk 1]
```diff
@@ -1670,6 +1670,8 @@ def update_integration_lists(new_integrations_names: List[str], packs_not_to_ins
     Returns:
         Tuple[List[str], List[str]]: The updated lists after filtering the turned non-hidden integrations.
     """
+    if not modified_integrations_names:
+        return new_integrations_names, modified_integrations_names
```
Comment (yaakovi) [inline Tests/configure_and_test_integration_instances.py:1674]:
```suggestion
        return new_integrations_names, modified_integrations_names

```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**